requirements/edx/github.txt

```diff
@@ -47,7 +47,6 @@
 -e git+https://github.com/cyberdelia/django-pipeline.git@1.5.3#egg=django-pipeline
 git+https://github.com/edx/django-wiki.git@v0.0.5#egg=django-wiki==0.0.5
 git+https://github.com/edx/django-openid-auth.git@0.8#egg=django-openid-auth==0.8
--e git+https://github.com/edx/django-rest-framework-oauth.git@f0b503fda8c254a38f97fef802ded4f5fe367f7a#egg=djangorestframework-oauth==1.0.1
```

Comment:
My understanding is that the other PR needed the uninstall step because the name of the kit was changing (to add the "edx-" prefix).  In this case, the name of the kit is not changing, so the uninstall should not be needed.